README.md

```diff
@@ -92,26 +92,75 @@ Two specific points to expand on:
 
 ### Setting up
 
-1. Follow the [Flutter installation guide](https://docs.flutter.dev/get-started/install)
-   for your platform of choice.
-2. Switch to the latest version of Flutter by running `flutter channel main`
-   and `flutter upgrade` (see [Flutter version](#flutter-version) below).
-3. Ensure Flutter is correctly configured by running `flutter doctor`.
-4. Start the app with `flutter run`, or from your IDE.
+1. Setup the pinned version of flutter.
```

Comment:
```suggestion
1. Set up the pinned version of Flutter.
```

(similar to https://github.com/zulip/zulip-flutter/pull/579#discussion_r1540337383)

There's also an occurrence of this in `.envrc`.